whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -342,7 +357,7 @@ private void mapEntityLinks(final Entity entity, final String requestUrl, final
                 throw new IllegalStateException("Malformed Url");
             }
             entity.getLinks().add(new Link(requestUrl, "self",
-                    url.getProtocol() + "://" + url.getHost() + "/" + entity.getObjectClassName() + "/" + attributeValue,
+                    url.getProtocol() + "://" + url.getHost() + PATH_DELIMITER + entity.getObjectClassName() + PATH_DELIMITER + attributeValue,
```

Comment:
I suggest either removing the `PATH_DELIMITER` constant (it doesn't help readability) and use String.format() instead OR (better I think) switch to `jakarta.ws.rs.core.UriBuilder` which is intended exactly for this purpose.

Reply:
Done